### PR TITLE
Clarify usage

### PR DESCRIPTION
**Pull request by mig4**
* add note about role required to be able to read organization info
* clarify format of `organization` argument to make it more obvious what values are expected

### CODE REVIEW

File: website/docs/d/google_organization.html.markdown [hunk 1]
```diff
@@ -22,13 +22,15 @@ resource "google_folder" "sales" {
...
+Note that the identity whose credentials are used for authentication needs to have `roles/resourcemanager.organizationViewer` role to be able to read that info.
+
 ## Argument Reference
 
 The arguments of this data source act as filters for querying the available Organizations.
 The given filters must match exactly one Organizations whose data will be exported as attributes.
 The following arguments are supported:
 
-* `organization` (Optional) - The name of the Organization in the form `{organization_id}` or `organizations/{organization_id}`.
+* `organization` (Optional) - The numeric ID of the Organization in the form `{organization_id}` or `organizations/{organization_id}`.
```
Comment (c2thorn) [inline website/docs/d/google_organization.html.markdown:33]:
```suggestion
* `organization` (Optional) - The name (numeric ID) of the Organization in the form `{organization_id}` or `organizations/{organization_id}`.
```
`name` is the actual term for the field here, as it is defined here: https://cloud.google.com/resource-manager/reference/rest/v1/organizations/get

So I'm hesitant to remove it completely. I understand the want for clarification, do you believe this suggestion will suffice?

**Comment by c2thorn** [comment]
I'm very sorry, this PR slipped through the cracks for me. I have a small suggestion, but otherwise looks fine.

If you want to, you can make the change upstream in Magic Modules
https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/website/docs/d/google_organization.html.markdown
And your change will propagate to both providers. No worries otherwise!

File: website/docs/d/google_organization.html.markdown (hunk 1 above)
Comment (mig4) [reply website/docs/d/google_organization.html.markdown:33]: Sure that makes sense. Initially I thought it means I can use either organization name or ID, that's why I thought it'd be good to clarify. Your suggestion works for me, thanks!

**Comment by mig4** [comment]
> I'm very sorry, this PR slipped through the cracks for me. I have a small suggestion, but otherwise looks fine.

That's all right of course. Thanks!

> If you want to, you can make the change upstream in Magic Modules

OK, will do.

**Comment by rileykarson** [comment]
Superseded with https://github.com/GoogleCloudPlatform/magic-modules/pull/4069

**Pull request closed without merging**